src/xdocs/beginning_development.xml

```diff
@@ -107,7 +107,7 @@
         </source>
       </div>
       <p>
-           7) Rebase your branch over your updated master
+           7) Rebase your branch over your updated master to share pulled updates
```

Comment:
`to share pulled updates` IMO doesn't make much sense. Rebase itself means the process of moving or combining a sequence of commits to a new base commit. So, I think the existing line pretty much says it all.